pkg/binary/envoy/controlplane/istio_test.go

```diff
@@ -32,42 +31,44 @@ import (
 	"github.com/tetratelabs/getenvoy/pkg/binary/envoytest"
 )
 
-func TestMain(m *testing.M) {
```

Comment:
I moved this into the test preconditions as it isn't necessary to do this for anything in the package